### PR TITLE
Prepare upstream sync

### PR DESCRIPTION
**Pull request by ced0ps**
Add CHANGELOG.md and upstream-sync script.
Add commit with [upstream-sync] flag for script to work

### CODE REVIEW

**Review by piontec — APPROVED** [review]
LGTMIWFY

File: hacks/helm-upstream-sync.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+fi
+
+curl -sL -o ${TMPDIR}/master.zip ${UPSTREAM_ZIP_URL}
+unzip -d ${TMPDIR}/ ${TMPDIR}/master.zip helm-charts-master/charts/jaeger-operator/*
+
+UPSTREAM_VERSION=$(cat ${TMPDIR}/helm-charts-master/charts/jaeger-operator/Chart.yaml | awk '/^version:/ { print $2 }')
+
+cp -a ${TMPDIR}/helm-charts-master/charts/jaeger-operator/* helm/jaeger-operator-app/
+
+git add helm/kong-app/
```
Comment (sslavic) [inline hacks/helm-upstream-sync.sh:30]:
```suggestion
git add helm/jaeger-operator-app/
```

Comment (ced0ps) [reply hacks/helm-upstream-sync.sh:30]: Thanks! Did not see that one.

File: hacks/helm-upstream-sync.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+curl -sL -o ${TMPDIR}/master.zip ${UPSTREAM_ZIP_URL}
+unzip -d ${TMPDIR}/ ${TMPDIR}/master.zip helm-charts-master/charts/jaeger-operator/*
+
+UPSTREAM_VERSION=$(cat ${TMPDIR}/helm-charts-master/charts/jaeger-operator/Chart.yaml | awk '/^version:/ { print $2 }')
+
+cp -a ${TMPDIR}/helm-charts-master/charts/jaeger-operator/* helm/jaeger-operator-app/
+
+git add helm/jaeger-operator-app/
+git commit -m "[upstream-sync] Version ${UPSTREAM_VERSION}"
+git rev-list --no-merges --reverse ${LAST_SYNC_COMMIT}..HEAD~1 -- helm/jaeger-operator-app/ | git cherry-pick -X theirs -n --stdin
```
Comment (sslavic) [inline hacks/helm-upstream-sync.sh:32]: Do you understand what this does?

Can it fail/conflict? How to proceed if it does?

**Review by sslavic — COMMENTED** [review]
LGTM

Curious, are we moving the app from playground to giantswarm catalog? I.e. is that rationale for preparing/working on getting it up-to-date?

**Pull request merged**